cumulus/outputs.tf

```diff
@@ -143,3 +143,11 @@ output "background_job_queue_url" {
 output "vpc" {
   value = data.aws_vpc.application_vpcs.id
 }
+
+output "cirrus_core_version" {
+  value = var.CIRRUS_CORE_VERSION
+}
+
+output "cirrus_daac_version" {
+  value = var.CIRRUS_DAAC_VERSION
+}
```

Comment:
Probably makes sense to do the same with the outputs, since they should also be present in every module

Reply:
They are at the end of the file. If we want them in a specific order, we should put them in alphabetical order. That should be done in a different PR. It is out of scope for this PR.

Reply:
One issue I see with putting them at the end of the file is that typically that's the default place where you're going to put new variables / outputs (as evidence by this PR!), so it can very quickly change from being at the end of the file to being 1 from the end of the file when a new variable is added. That's why I think for stuff that is super core and unlikely to change, it should go at the top. 

For stuff that's all mostly at the same level of importance, an alphabetically organized section would make sense too. But I would consider the cirrus versions, deploy name, and maturity to all be more important than the rest, and deserve to be put somewhere where they are unlikely to be bumped around as the files churn. I think the top makes the most sense.